packages/app-next/package.json

```diff
@@ -62,7 +62,6 @@
     "react-router": "^6.3.0",
     "react-router-dom": "^6.3.0",
     "react-use": "^17.2.4",
-    "vite": "^4.4.9",
     "vite-plugin-html": "^3.2.0",
```

Comment:
Does it make sense to remove all the deps?

Reply:
UPDATED